TwilioVerifyDemo/TwilioVerifyDemo/CreateFactor/Presenter/CreateFactorPresenter.swift

```diff
@@ -31,14 +31,9 @@ class CreateFactorPresenter {
   private let accessTokensAPI: AccessTokensAPI
   
   init?(withView view: CreateFactorView, accessTokensAPI: AccessTokensAPI = AccessTokensAPIClient()) {
```

Comment:
I think you can remove the optional from the init

Reply:
Added a validation for the twilio verify dependency